lua/deferred-clipboard.lua

```diff
@@ -7,17 +7,12 @@ local function is_continuous_clipboard_sync_enabled()
         and vim.o.clipboard ~= ''
 end
 
-local function schedule_disable_of_continuous_clipboard_sync_on_focus_change()
-    vim.api.nvim_create_autocmd({
-        'FocusGained',
-        'FocusLost',
-    }, {
-        once = true,
-        pattern = '*',
-        callback = function()
-            vim.o.clipboard = nil
-        end,
-    })
+local function disable_continuous_clipboard_sync()
+    local hasFocusLostEvent = vim.api.nvim_command_output(([[echo exists('##FocusLost')]]))
+    local hasFocusGainedEvent = vim.api.nvim_command_output(([[echo exists('##FocusGained')]]))
+    if hasFocusLostEvent == '1' or hasFocusGainedEvent == '1' then
```

Comment:
After the change above we don't need to compare with `1` anymore here, but if we have a reliable way to check for these events support we should only disable continuous clipboard if both of them are supported, so use `and` instead of `or`.

```suggestion
    if is_focus_lost_event_supported and is_focus_gained_event_supported then
```